src/App/Fossa/Analyze/Project.hs

```diff
@@ -25,14 +26,17 @@ mkResult project graphResults =
         if S.null (Graphing.graphingDirect graph)
           then graph
           else Graphing.pruneUnreachable graph
-    , projectResultGraphBreadth = graphBreadth
+    , projectResultGraphBreadth = dependencyGraphBreadth dependencyResults
+    , projectResultManifestFiles = relativeManifestFiles
     }
   where
-    (graph, graphBreadth) = graphResults
+    graph = dependencyGraph dependencyResults
+    relativeManifestFiles = map (toRelativePath basedir) $ dependencyManifestFiles dependencyResults
 
 data ProjectResult = ProjectResult
   { projectResultType :: Text
   , projectResultPath :: Path Abs Dir
   , projectResultGraph :: Graphing Dependency
   , projectResultGraphBreadth :: GraphBreadth
+  , projectResultManifestFiles :: [FilePath]
```

Comment:
the `path` library we use introduced a new type called `SomeBase`:

```hs
data SomeBase t
  = Abs (Path Abs t)
  | Rel (Path Rel t)
```

..which would allow us to more directly specify what's in this field: absolute and relative files.

```suggestion
  , projectResultManifestFiles :: [SomeBase File]
```